src/components/ellenTicTacToe.test.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import { render, shallow, mount } from 'enzyme';
+import Game from './tictactoe';
+describe('the status', () => {

```

Comment:
Good tests. It'd be nice to extract some variables for the shared data, or maybe a function that generates it? That'd also allow you to give a name to the data to make it explicit what's notable about the state (`neutralState`, `winningState`, `tieState`, etc.).